fsharp-backend/src/LibExecutionStdLib/LibDate.fs

```diff
@@ -256,6 +256,20 @@ let fns : List<BuiltInFn> =
 
 
     { name = fn "Date" "subtract" 0
+      parameters = [ Param.make "d" TDate ""; Param.make "seconds" TInt "" ]
```

Comment:
This should use the rename functionality: https://github.com/darklang/dark/blob/main/fsharp-backend/src/LibExecutionStdLib/StdLib.fs#L10

Reply:
Ahh, OK. When you mentioned renames previously, I thought you were referring to `Deprecation.RenamedTo`. Thanks